packages/tracing/src/browser/metrics.ts

```diff
@@ -154,6 +182,14 @@ export class MetricsInstrumentation {
             ...(entry.size && { elementSize: entry.size }),
             value: entry.startTime,
           };
+
+          logger.log('[Measurements] Adding LCP (Largest Contentful Paint)');
+
+          this._measurements['lcp'] = { value: entry.startTime };
+
+          const timeOrigin = msToSec(performance.timeOrigin);
+          const startTime = msToSec(entry.startTime as number);
+          this._measurements['mark.lcp'] = { value: timeOrigin + startTime };
```

Comment:
Purely stylistic suggestion, only reordering lines, to make this block consistent with the other similar occurrences (log message, set measurement X, set mark.X):

```suggestion
          const timeOrigin = msToSec(performance.timeOrigin);
          const startTime = msToSec(entry.startTime as number);
          logger.log('[Measurements] Adding LCP (Largest Contentful Paint)');
          this._measurements['lcp'] = { value: entry.startTime };
          this._measurements['mark.lcp'] = { value: timeOrigin + startTime };
```